modules/ROOT/pages/migrate-helm.adoc

```diff
@@ -0,0 +1,62 @@
+== Migrating to Helm Managed Runtime Fabric
+
+You can migrate to xref:runtime-fabric::install-helm.adoc[helm] native for Runtime Fabrics that were installed using xref:runtime-fabric::install-self-managed.adoc[rtfctl].
+
+== Before You Begin
+
+To perform the migration, you must consider the following:
+
+* The migration is applicable to BYOK clusters which are installed with `rtfctl` and want to migrate to helm managed.
+* The cluster must be on `rtf-agent` v2.2.4 or above. Note that you will first have to upgrade latest before you can initiate the migration. 
+
+Additionally, you must have these details:
+
+* organizationId
+* fabricId
+* Bearer token for Anypoint user account with *manage fabric* privileges
+* rtfctl version >= v1.0.10 is downloaded
+
+== Migrate rtfctl to Helm
+
+To migrate your Runtime Fabric from rtfctl to helm, run this command:
+
+[source,copy]
+----
+rtfctl migrate previous-rtf-bkp.tar.gz --organizationId <orgId value> --fabricId <fabricId value> --token <bearer token> --host qax.anypoint.mulesoft.com  --confirm
+----
+
+Running this command performs the following actions:
+
+* Backs up the current Runtime Fabric installation: tarball is created with input file name.
+* Uninstalls the current Runtime Fabric installation: running the command does not update Mule application deployments, it does not delete any of your configurations applied to the rtf namespace, it does not delete the rtf namespace, and it does not touch any other namespace than the rtf namespace. Uninstalls only the previous runtime-fabric helm chart.
+* Regenerates activation data for provided fabricId.
+* Deletes custom-properties secret in the rtf namespace, because in helm managed rtf the secret is recreated using values from the helm chart.
```

Comment:
```suggestion
* Deletes custom-properties secret in the rtf namespace, because helm managed rtf recreates the secret using values from the helm chart.
```